### ISSUE TITLE
Web -  Refreshing the editing photo section will make the process to go back

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#22936]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to settings > Profile
2. Click the avatar and then the "upload photo" Option
3. Choose the photo you want to upload and refresh

## Expected Result:
It should still be in the "Edit Photo" section.

## Actual Result:
Refreshing the section will make it go back to the profile section

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.40-5
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/b97112f5-2aff-42a8-9c4c-5074c76d9bc2

https://github.com/Expensify/App/assets/93399543/984c07e3-a7e9-4c6f-8671-ce1aaa974f95

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Yokabdk
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689344116454879

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by twisterdotcom** [comment]
I agree. I think auto-saving is a feature, and not doing that would be a regression actually.

**Comment by flaviadefaria** [comment]
We decided to close this for now.
